lib/mix/phoenix.ex

```diff
@@ -65,6 +65,47 @@ defmodule Mix.Phoenix do
   defp to_app_source(app, source_dir) when is_atom(app),
     do: Application.app_dir(app, source_dir)
 
+  @default_format_extensions [".ex", ".exs", ".heex"]
+  @doc """
+  Conditionally run `mix format` for generated files.
+  By default, unless `format_extensions` is set in generators config,
+  files with extensions `#{inspect(@default_format_extensions)}` are formatted and
+  override format instruction is printed into console.
+  `format_extensions` can be set to `[]` to turn off formatting.
+  If no files pass condition, formatting is not performed.
+  """
+  def maybe_format(files) when is_list(files) do
+    config = Application.get_env(otp_app(), :generators, []) |> Keyword.get(:format_extensions)
+    format_extensions = config || @default_format_extensions
+    files = Enum.filter(files, &String.ends_with?(&1, format_extensions))
+
+    if files != [] do
+      Mix.Task.run("format", files)
+      if !config, do: Mix.shell().info(override_format_instruction())
```

Comment:
We don't need to print this every time we run. Users will read the docs if they need it. Especially because we only format the files we only generate.

Reply:
Sure, deleted console notification.